wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/DevfileService.java

```diff
@@ -124,16 +119,8 @@ public Response createFromYaml(String data)
       throws ServerException, ConflictException, NotFoundException, ValidationException,
           BadRequestException {
 
-    WorkspaceImpl workspace;
-    try {
-      DevfileImpl devfile = devfileManager.parseYaml(data);
-      workspace = devfileManager.createWorkspace(devfile, urlFileContentProvider);
-    } catch (DevfileException e) {
-      throw new BadRequestException(e.getMessage());
-    }
-    return Response.status(201)
-        .entity(asDto(workspace).withLinks(linksGenerator.genLinks(workspace, getServiceContext())))
-        .build();
+    throw new BadRequestException(
+        "This is an outdated method please use /api/workspace/devfile to create a new workspace");
```

Comment:
Run-on sentence and does not explain WHY user should use other api.

Preferred: "This method is deprecated and may be removed at any time. Please use ..."